frontend/src/app/main/ui/settings.cljs

```diff
@@ -55,5 +56,8 @@
          [:& password-page {:locale locale}]
 
          :settings-options
-         [:& options-page {:locale locale}])]]]))
+         [:& options-page {:locale locale}]
+
+         :settings-access-tokens
+         [:& access-tokens-page {:locale locale}])]]]))
```

Comment:
Don't forward locale in the same way as options-page, it is deprecated and should be removed completely